cortx-monitorQuickstartGuide.md

```diff
@@ -137,23 +137,83 @@ This guide provides a step-by-step walkthrough for getting you CORTX-Monitor Ser
     -rw-r--r-- 1 root root   9592 Aug 18 07:27 cortx-libsspl_sec-method_pki-1.0.0-1_git8907300.el7.x86_64.rpm
     -rw-r--r-- 1 root root 101004 Aug 18 07:27 cortx-libsspl_sec-devel-1.0.0-1_git8907300.el7.x86_64.rpm
     ```
-
-3. Install CORTX-Monitor RPMs
+    
+5. Install CORTX-Monitor RPMs
 
    You'll need to create a repository and copy all the required RPMs to it. Run the commands:
 
     ```shell
-    
     $ mkdir MYRPMS
     $ cp -R /root/rpmbuild/RPMS/x86_64/cortx-libsspl_sec-* /root/rpmbuild/RPMS/noarch/cortx-sspl-* MYRPMS
-    $ find MYRPMS -name \*.rpm -print0 | sudo xargs -0 yum install -y
+    $ CORTX_MONITOR_BASE_URL="https://raw.githubusercontent.com/Seagate/cortx-monitor/main"
+    $ curl $CORTX_MONITOR_BASE_URL/low-level/files/opt/seagate/sspl/setup/sspl_dev_deploy -o sspl_dev_deploy
+    $ chmod a+x sspl_dev_deploy
+    ```
+    **sspl_dev_deploy is only meant for dev purpose and to install SSPL independently along with its dependencies.**
+    ```
+    $ ./sspl_dev_deploy --cleanup
```

Comment:
`--cleanup` includes removing cortx-py-utils RPM.
**Caution**: Any other component RPMS installed on the setup also will get removed while removing cortx-py-utils.

Reply:
added a note below the command.